db/seeds.rb

```diff
@@ -181,28 +182,30 @@ def position(type)
     :common_name => surname,
     :position => position,
     :premier_league_team_id => player['team'],
+    :picture => picture,
     :gameweek_points => { 1 => 0 },
+    
   )
 
   puts "done #{full_name}" if (player['id'].to_i % 50).zero?
 end
 
 # player team relationships
 starting_team_relationships = {
-  '14' => [93, 141, 401, 182, 518, 192, 431, 151, 217, 67, 409], # AD1
-  '15' => [111, 122, 274, 309, 358, 321, 463, 119, 154, 210, 363], # CT
-  '16' => [212, 123, 104, 486, 343, 171, 138, 369, 31, 12, 91], # DW
-  '17' => [388, 161, 108, 459, 143, 214, 354, 399, 17, 523, 496], # JK
-  '18' => [235, 185, 181, 103, 184, 207, 113, 29, 346, 114, 166], # JT
-  '19' => [262, 206, 337, 144, 28, 265, 390, 292, 187, 36, 68], # KW
-  '20' => [14, 202, 203, 515, 226, 193, 15, 215, 239, 457, 11], # NM
-  '21' => [189, 252, 3, 159, 191, 266, 320, 368, 90, 466, 460], # PB
-  '22' => [366, 183, 332, 186, 218, 116, 220, 414, 415, 487, 167], # PL
-  '23' => [198, 331, 106, 405, 344, 176, 199, 219, 278, 211, 437], # XL
-  '24' => [94, 182, 381, 258, 325, 442, 488, 525,283, 338, 428], # AD2
-  '25' => [340, 105, 2, 160, 150, 238, 342, 134, 133, 188, 313], # AL
-  '26' => [411, 330, 179, 59, 75, 448, 392, 469, 50, 233, 44], # GM
-  '27' => [168, 42, 163, 39, 10, 24, 393, 244, 347, 365, 175], # HT
+  '1' => [93, 141, 401, 182, 518, 192, 431, 151, 217, 67, 409], # AD1
```

Comment:
why did you change all these numbers?

Reply:
It was giving me the wrong (last year's) teams, I think it must be different in live?